src/hidimstat/desparsified_lasso.py

```diff
@@ -4,141 +4,111 @@
-        Method used for computing the residuals of the Nodewise Lasso.
-        Currently the only method available is 'lasso'.
+        Convergence tolerance for optimization.
 
     alpha_max_fraction : float, optional (default=0.01)
-        Only used if method='lasso'.
-        Then alpha = alpha_max_fraction * alpha_max.
+        Fraction of max alpha used for Lasso regularization.
 
-    n_jobs : int or None, optional (default=1)
-        Number of CPUs to use during the Nodewise Lasso.
+    eps : float, optional (default=1e-2)
+        Small constant used in noise estimation.
+
+    tol_reid : float, optional (default=1e-4)
+        Tolerance for Reid estimation.
+
+    n_split : int, optional (default=5)
+        Number of splits for cross-validation in Reid procedure.
+
+    n_jobs : int, optional (default=1)
+        Number of parallel jobs. Use -1 for all CPUs.
+
+    seed : int, default=0
+        Random seed for reproducibility.
+
+    verbose : int, default=0
+        Verbosity level for logging.
 
-    memory : str or joblib.Memory object, optional (default=None)
-        Used to cache the output of the computation of the Nodewise Lasso.
-        By default, no caching is done. If a string is given, it is the path
-        to the caching directory.
+    group : bool, default=False
+        If True, use group Lasso for multiple responses.
 
-    verbose: int, optional (default=1)
-        The verbosity level: if non zero, progress messages are printed
-        when computing the Nodewise Lasso in parralel.
-        The frequency of the messages increases with the verbosity level.
+    cov : ndarray, shape (n_times, n_times), default=None
+        Temporal covariance matrix of the noise.
+        If None, it is estimated.
+
+    noise_method : {'AR', 'simple'}, default='AR'
+        Method to estimate noise covariance:
+        - 'simple': Uses median correlation between consecutive
+                    timepoints
+        - 'AR': Fits autoregressive model of specified order
+
+    order : int, default=1
+        Order of AR model when noise_method='AR'. Must be < n_times.
+
+    fit_Y : bool, default=True
+        Whether to fit Y in noise estimation.
+
+    stationary : bool, default=True
+        Whether to assume stationary noise in estimation.
 
     Returns
     -------
-    beta_hat : array, shape (n_features,)
-        Estimated parameter vector.
+    beta_hat : ndarray, shape (n_features,) or (n_features, n_times)
+        Desparsified Lasso coefficient estimates.
 
-    cb_min : array, shape (n_features)
-        Lower bound of the confidence intervals on the parameter vector.
+    sigma_hat/theta_hat : float or ndarray, shape (n_times, n_times)
+        Estimated noise level (single response) or precision matrix
+        (multiple responses).
 
-    cb_max : array, shape (n_features)
-        Upper bound of the confidence intervals on the parameter vector.
+    omega_diag : ndarray, shape (n_features,)
```

Comment:
>I'm confused. Is this the covariance or precision ?
https://github.com/mind-inria/hidimstat/pull/156#discussion_r1964371916

What do @bthirion think?

Reply:
@bthirion

Reply:
It is the precision. Actually, it should be renamed from `omega_diag` to `precision_diag`.

Reply:
done